src/polylineencoder.h

```diff
@@ -242,6 +245,32 @@ std::string PolylineEncoder<Digits>::encode(const typename PolylineEncoder::Poly
     return result;
 }
 
+template <int Digits>
+template <typename T>
+std::string PolylineEncoder<Digits>::encodeList(const T *data, size_t list_length, double T::*latitude, double T::*longitude)
```

Comment:
This function almost completely duplicates the existing `PolylineEncoder::encode()` one. We must avoid the code duplication.

Reply:
the inner part of the function is the same as you mentioned, should I put the repeated part in a new function?